finger-exercises/redux/src/app/screens/App/index.js

```diff
@@ -58,21 +57,52 @@ class App extends Component {
         <Navbar />
         <div className={styles.container}>
           <Search onSearch={this.onSearch} />
-          {this.state.books.length ? (
-            this.state.books.map(this.renderBooks)
+          {this.props.books.length ? (
```

Comment:
Remember to destructure props before the return statement.
Destructuring before usinf props or state, favours readability and clarity to understand what the component is rendering.